src/base/similarity_transform.cc

```diff
@@ -194,6 +196,19 @@ SimilarityTransform3::SimilarityTransform3(const double scale,
   transform_.matrix() = matrix;
 }
 
+SimilarityTransform3::SimilarityTransform3(const std::string& path) {
```

Comment:
Seems like this is redundant to the file reading functionality from the model_cropper/splitter PR?

Reply:
Yes it is. I did not want to make dependent PRs so I included the change here as well. I will change to the static `FromFile` as agreed in the other PR #1179 and resolve the conflict (if it appears) in whichever gets merged last.